velox/dwio/dwrf/common/RLEv2.cpp

```diff
@@ -206,7 +206,6 @@ void RleDecoderV2<isSigned>::next(
 
     if (runRead == runLength) {
       resetRun();
-      firstByte = readByte();
```

Comment:
Remove the `enc` below and use `type` instead

Reply:
Done